Docker/dev.pgsql.Dockerfile

```diff
@@ -24,6 +24,9 @@ COPY --from=assets ./Application/EdFi.Ods.AdminApi.AdminConsole EdFi.Ods.AdminAp
 COPY --from=assets ./Application/NuGet.Config EdFi.Ods.AdminApi.Common/
 COPY --from=assets ./Application/EdFi.Ods.AdminApi.Common EdFi.Ods.AdminApi.Common/
 
+COPY --from=assets ./Application/NuGet.Config EdFi.AdminConsole.HealthCheckService/
+COPY --from=assets ./Application/EdFi.AdminConsole.HealthCheckService EdFi.AdminConsole.HealthCheckService/
+
```

Comment:
These are frustrating false positives. I wish Hadolint would update to recognize the new syntax for additional contexts.